src/CAPI/include/FMI3/fmi3_capi.h

```diff
@@ -728,14 +728,14 @@ fmi3_status_t fmi3_capi_completed_integrator_step(fmi3_capi_t* fmu,
     fmi3_boolean_t* enterEventMode, fmi3_boolean_t* terminateSimulation);
 
 /**
- * \brief Calls the FMI function fmiGetDerivatives(...)
+ * \brief Calls the FMI function fmiGetContinuousStateDerivatives(...)
  *
  * @param fmu C-API struct that has succesfully loaded the FMI function.
- * @param derivatives (Output) Array of the derivatives.
- * @param nx Number of derivatives.
+ * @param derivatives (Output) Array of the continuous state derivatives.
+ * @param nx Number of continuous state derivatives.
  * @return FMI status.
  */
-fmi3_status_t fmi3_capi_get_derivatives(fmi3_capi_t* fmu, fmi3_float64_t derivatives[], size_t nx);
+fmi3_status_t fmi3_capi_get_continuous_state_derivatives(fmi3_capi_t* fmu, fmi3_float64_t continuousStateDerivatives[], size_t nx);
```

Comment:
Good you found this.

We probably want the args to match here and in the impl (.c file), so I suggest naming continuousStateDerivatives just `derivatives` again. It's quite clear what it is, and it's the same as in the standard header

Reply:
Changed the arg to derivatives for consistency (and shorter lines)